src/core/package_handling.cpp

```diff
@@ -481,10 +480,10 @@ namespace mamba
         }
         catch (...)
         {
-            LOG_WARNING << "Could not read paths.json from " << pkg_folder << std::endl;
```

Comment:
not sure that trace is the correct replacement for WARNING

Reply:
this method is called here https://github.com/mamba-org/mamba/blob/42475c057869356f6652e9ef9ba7f94f520567fc/src/core/package_cache.cpp#L226 and the warning will be emitted.
This is not necessary to emit 2 warnings for the same invalid cache, so I put the details in `trace`

Reply:
this stuff almost _never_ happens and when it happens it would help us to locate a bug more easily since the users would see the (more precise) warnings right away. So it could help us with future bugs